src/System.Runtime.Extensions/src/System/AppDomain.cs

```diff
@@ -27,4 +27,19 @@ public partial class AppDomain
             }  
         }  
     }
-}
\ No newline at end of file
+}
+namespace System.Runtime.ExceptionServices
+{
+    public class FirstChanceExceptionEventArgs : EventArgs
+    {
+        private Exception m_Exception;
+        public FirstChanceExceptionEventArgs(Exception exception)
+        {
+             m_Exception = exception;
+        }
+        public Exception Exception
+        {
+            get { return m_Exception; }
+        }
```

Comment:
The first chance exception handling needs to be wired into CoreLib. It would be more natural to leave FirstChanceExceptionEventArgs in CoreLib - so that it is symmetric with UnhandledExceptionEventArgs, and do not have to do ugly argument remapping.